integrations/zendesk/src/definitions/index.ts

```diff
@@ -0,0 +1,54 @@
+import type { IntegrationDefinitionProps } from '@botpress/sdk';
+import { z } from 'zod';
+
+export { actions } from './actions';
+export { events } from './events';
+export { channels } from './channels';
+
+export const configuration = {
+  schema: z.object({
+    baseURL: z.string({
+      description:
+        'Your zendesk organization URL. e.g. https://{subdomain}.zendesk.com',
+    }),
+    username: z.string({
+      description:
+        'Your zendesk account email, add "/token" to the end. e.g. jdoe@example.com/token',
+    }),
+    apiToken: z.string({
+      description: 'Zendesk API Token',
+    }),
+  }),
+};
+
+export const states: IntegrationDefinitionProps['states'] = {
+  subscriptionInfo: {
+    type: 'integration',
+    schema: z.object({
+      subscriptionId: z.string(),
+    }),
+  },
+  ['trigger0ticketAssigned']: {
```

Comment:
<!-- Questions are appropriate if you have a potential concern but are not quite sure if it's relevant or not. Asking the author for clarification or investigation can lead to a quick resolution.  -->
**❔ question:** ‏ Why is the state name containing `0` ? Could the state be named `triggerTicketAssigned` instead ?